packages/router/src/grpc/custody/view-transaction-plan/view-action-plan.ts

```diff
@@ -94,6 +100,51 @@ const getOutputView = (
   });
 };
 
+const getSwapView = (swapPlan: SwapPlan): SwapView => {
```

Comment:
This, and `getSwapClaimView` below it, are quite important functions, since they determine what we'll be showing the user when they decide whether to approve or deny a transaction. Please review these thoroughly to make sure they're accurately copying over from Plan -> View

Reply:
swap claims and views are working properly for me